napari/_vispy/text_utils.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+from vispy.scene.visuals import Text
+
+
+def update_text(
+    text_values: np.ndarray,
+    text_coords: np.ndarray,
+    anchor_x: str,
+    anchor_y: str,
```

Comment:
Can this be `anchor` a 2-tuple of str?

Reply:
I tend to prefer explicit kwargs that take a single value since I think it is simpler to remember (e.g., one doesn't have to remember what each element in the tuple is). In this particular case, it's only used internally and we would always provide both values, so I don't have a strong preference. I went ahead and made the change!